CRM/Core/I18n/Schema.php

```diff
@@ -188,18 +188,20 @@ public static function makeSinglelingualTable(
       }
     }
 
+    $dao = new CRM_Core_DAO();
     // deal with columns
     foreach ($columns[$table] as $column => $type) {
-      $queries[] = "ALTER TABLE {$table} ADD {$column} {$type}";
-      $queries[] = "UPDATE {$table} SET {$column} = {$column}_{$retain}";
+      $queries[] = "ALTER TABLE {$table} CHANGE `{$column}_{$retain}` `{$column}` {$type}";
       foreach ($locales as $loc) {
-        $dropQueries[] = "ALTER TABLE {$table} DROP {$column}_{$loc}";
+        if (strcmp($loc,$retain) !== 0) {
```

Comment:
Yep - it seems we do still support 5.1 https://wiki.civicrm.org/confluence/display/CRMDOC/Installation+and+Upgrades